jquery.validation/index.d.ts

```diff
@@ -98,14 +109,21 @@ declare namespace JQueryValidation
          */
         onclick?: ShouldValidatePredicate;
         /**
+         * Function. Validate elements when user focues in. If ommited hides all other filed marked as invalid. 
```

Comment:
1. `onkeyup` doesn't allow Boolean in its type, and documents it. `onfocusin` should too.
2. typos: focusses, omitted, files

Of course, it might be better to add a different type that is just the function and doesn't allow boolean.